src/garage/np/exploration_policies/epsilon_greedy_policy.py

```diff
@@ -42,9 +42,10 @@ def __init__(self,
         self._min_epsilon = min_epsilon
         self._decay_period = int(total_timesteps * decay_ratio)
         self._action_space = env_spec.action_space
-        self._epsilon = self._max_epsilon
         self._decrement = (self._max_epsilon -
                            self._min_epsilon) / self._decay_period
+        self._total_env_steps = 0
+        self._last_total_env_steps = 0
```

Comment:
Is this field needed? It only seems to be set, not used.

Reply:
I use it in `update` method. Every time we update the `total_env_step`, we only need add the steps of new batch into this `last_total_env_steps`, so that we can update the `total_env_step` each step in one batch and do not infect the cumulative env steps.